include/nanobind/nb_lib.h

```diff
@@ -19,10 +19,9 @@ struct NB_CORE cleanup_list {
 public:
     static constexpr uint32_t Small = 6;
 
-    cleanup_list(PyObject *self) {
-        m_size = 1;
-        m_capacity = Small;
-        m_data = m_local;
+    cleanup_list(PyObject *self) : m_size{1},
+		   m_capacity{Small},
```

Comment:
I am curious -- what's the point of brace init `{` here as opposed to the IMO more readable parenthesis-based init? I use the former mainly for initializing POD `struct` when really necessary (when they don't have a constructor). However, here it would seem that both are equally usable.

Reply:
Main difference is that it prevents narrowing conversions I believe and is the recommended way to initialize types in C++11 or higher.